### ISSUE TITLE
"please wait whilst data is loading" error

### ISSUE THREAD

**Issue by derkoi** [yuliskov/SmartTubeNext#1417]
Since updating my Nvidia shield TV today I've noticed I have issues when trying to open a channel, subscribe, open the video description or the end of a video watched I get the message "please wait whilst data is loading" and then nothing happens. This is from the menu when I hold the OK button down on a video.

Not sure why it's happening, I've tried uninstall ING and installing the beta version and it's the same.

**Comment by ncaa380** [comment]
Same problem here as well. Also I can't skip to the next video. Note: if I log out it starts working as intended, but when I log back in it keeps giving the "please wait while data is loading" message over and over.

**Comment by koffeedude** [comment]
I have the same issue.. I am using on Firecube and it started with the new update of smarttubenext. I also logged out everything works but once I log in it starts acting up!  Very strange! I want to stay logged in to access my subscriptions and other account playlists!

**Comment by koffeedude** [comment]
Update 14.60 solved the problem for me. This update was just added today and now everything seems to be working again.

**Comment by koffeedude** [comment]
Update 14.60 solved the problem for me. This update was just added today.. most of my subscription links is working again.

**Comment by derkoi** [comment]
> Update 14.60 solved the problem for me. This update was just added today.. most of my subscription links is working again.

Yes, I just updated and it seems to have fixed the issues I was having too. Excellent work devs. 👍🏻
